ferrilata-devilline/Startup.cs

```diff
@@ -1,27 +1,37 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using Microsoft.Extensions.Configuration;
 using ferrilata_devilline.Repositories;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
+using System;
 
 namespace ferrilata_devilline
 {
     public class Startup
     {
+
+        private IConfiguration Configuration;
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
 
+            string connection = Environment.GetEnvironmentVariable("ConnectionStringsDevillineDB");
+
             services.AddDbContext<ApplicationContext>(builder => builder
-            .UseMySQL("server=localhost;database=EFCoreTodo;user=root;password="));
+            .UseMySQL($"server={Environment.GetEnvironmentVariable("FDHOST")} " +
+            $"database={Environment.GetEnvironmentVariable("FDDATABASE")} " +
+            $"user={Environment.GetEnvironmentVariable("FDUSERNAME")}" +
+            $" password={Environment.GetEnvironmentVariable("FDPASSWORD")}"));
```

Comment:
It's unclear where your variables are set up, since you specify a connection string (line 28) but then you build it again in here (lines 31-34)

Reply:
Env vars are coming from System, they are set in terminal